core/src/components/label/label.md.scss

```diff
@@ -15,13 +15,17 @@
 :host(.label-stacked) {
   @include transform-origin(start, top);
   @include transform(translate3d(0, 50%, 0), scale(.75));
+
+  transition: color 150ms cubic-bezier(.4,0,.2,1);
 }
 
 :host(.label-floating) {
   @include transform(translate3d(0, 96%, 0));
   @include transform-origin(start, top);
 
-  transition: transform 150ms cubic-bezier(.4,0,.2,1);
+  transition:
+    color 150ms cubic-bezier(.4,0,.2,1),
```

Comment:
We might want to make this cubic-bezier a scss var since it's used a few times here